shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java

```diff
@@ -42,6 +46,12 @@ public abstract class AbstractShenyuPlugin implements ShenyuPlugin {
 
     private static final Logger LOG = LoggerFactory.getLogger(AbstractShenyuPlugin.class);
 
+    private static ShenyuConfig.MatchCache matchCacheConfig;
+
+    private static SelectorData nullSelectorData = new SelectorData();
+
+    private static RuleData nullRuleData = new RuleData();
```

Comment:
Oh, `nullSelectorData` and `nullRuleData`, maybe shouldn not use `static` modifer?